src/core_plugins/vega/public/default.spec.hjson

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
imho, I'd remove this. The initial default is nice, but it only works if there's a corresponding index, which is unlikely.

There's already some nice links to the documentation, that can include a little tutorial with makelogs data, but now this just isn't functional out of the box.

Reply:
per slack, keeping the default - it uses _all index - similar to timelion